src/core/entities/crux-user.ts

```diff
@@ -63,6 +63,7 @@ export class CruxUser {
     private cruxUserConfig!: ICruxUserConfiguration;
     private cruxUserPrivateAddresses!: ICruxUserPrivateAddresses;
     private cruxDomain!: CruxDomain;
+    private blacklistedCruxIDs!: string[];
```

Comment:
remove all changes in this file. we are not using this feature right now.

Reply:
done and updated related places as well